include/CRADLE/Utilities.hh

```diff
@@ -496,9 +627,27 @@ namespace utilities {
       int betaType = (int)((Z > 0) - (Z < 0));
       Z = std::abs(Z);
       double cShape, cNS;
-      std::tie(cShape, cNS) = CCorrectionComponents(W, W0, Z, A, R, betaType, GAMOW_TELLER, 1.27, -229, 1, 4.*A, 1*A, 0);
+
+      if (dm.configOptions.betaDecay.Default == "Fermi") {
```

Comment:
It's good practice to define the decayType variable before the if statements, and only change it when needed. This avoids potentially undefined variables when changing something later.